### PR TITLE
Add calendar extension layer

### PR DESCRIPTION
**Pull request by semsphy**
Add calendar extension.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ functions:
 |:-------------------|:--------------------------------------|
 | AMQP               | `${bref:extra.amqp-php-74}`           |
 | Blackfire          | `${bref:extra.blackfire-php-74}`      |
+| Calendar          | `${bref:extra.calendar-php-74}`      |
```
Comment (GrahamCampbell) [inline Readme.md:45]: missing spaces ;)

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 1 above)
Comment (semsphy) [reply Readme.md]: Nice catch :)

File: Readme.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ functions:
 |:-------------------|:--------------------------------------|
 | AMQP               | `${bref:extra.amqp-php-74}`           |
 | Blackfire          | `${bref:extra.blackfire-php-74}`      |
+| Calendar          | `${bref:extra.calendar-php-74}`        |
```
Comment (GrahamCampbell) [inline Readme.md:45]: Alignment still incorrect.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 2 above)
Comment (semsphy) [reply Readme.md:45]: :( thank.

**Comment by semsphy** [comment]
@Nyholm, any chance to review my PR, pls.

File: layers/calendar/config.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "php": [
+        "73",
+        "74",
+        "80",
```
Comment (Nyholm) [inline layers/calendar/config.json:5]: This is not valid json. 

```suggestion
        "80"
```

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**